.github/workflows/publish-artifacts.yml

```diff
@@ -29,3 +32,24 @@ jobs:
           docker tag "${DOCKERHUB_WORKSPACE}/${UI_IMAGE_REPO}:latest" "${DOCKERHUB_WORKSPACE}/${UI_IMAGE_REPO}:${{ github.event.release.tag_name }}"
           docker push "${DOCKERHUB_WORKSPACE}/${UI_IMAGE_REPO}:${{ github.event.release.tag_name }}"
           docker push "${DOCKERHUB_WORKSPACE}/${UI_IMAGE_REPO}:latest"
+
+  python-packages:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Generate SDK
+        run: |
+          pip3 install --user -r cvat-sdk/gen/requirements.txt
+          ./cvat-sdk/gen/generate.sh
+
+      - name: Build packages
+        run: |
+          for d in cvat-sdk cvat-cli; do
+            pipx run --spec=build pyproject-build --outdir=dist "$d"
+          done
+
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Have you already set up PyPI project?

Reply:
No, I was going to do that after this PR was merged.

Reply:
And now I did it.